entityDumper/configurationDumper.go

```diff
@@ -0,0 +1,123 @@
+package entityDumper
+
+import (
+	"bufio"
+	"database/sql"
+	"fmt"
+	"github.com/rs/zerolog/log"
+	"github.com/uyuni-project/inter-server-sync/dumper"
+	"github.com/uyuni-project/inter-server-sync/dumper/packageDumper"
+	"github.com/uyuni-project/inter-server-sync/schemareader"
+	"github.com/uyuni-project/inter-server-sync/sqlUtil"
+	"os"
+)
+
+func ConfigTableNames() []string {
+	return []string{
+		"rhnconfigfile",
+		"rhnconfigfilename",
+		"rhnconfigrevision",
+		"rhnconfigcontent",
+		"rhnconfigchannel",
+		"rhnconfigfilestate",
+		"rhnregtokenconfigchannels",
+		"rhnserverconfigchannel",
+		"rhnsnapshotconfigchannel",
+		"susestaterevisionconfigchannel",
+		"rhnconfiginfo",
+		"rhnconfigfilefailure",
+		"rhnchecksum",
+		"rhnchecksumtype",
+		"web_contact",
+	}
+}
+
+func DumpConfigs(options ChannelDumperOptions) {
+	var outputFolderAbs = options.GetOutputFolderAbsPath()
+	db := schemareader.GetDBconnection(options.ServerConfig)
+	defer db.Close()
+	//file, err := os.OpenFile(outputFolderAbs + "/sql_statements.sql", os.O_APPEND | os.O_WRONLY | os.O_CREATE, 0600)
```

Comment:
this line is commented and it would solve the question to append to a single file. We should use it or remove it.